11-information_governance.Rmd

```diff
@@ -28,187 +28,119 @@ If requesting access to a data source, you should provide the GitHub usernames o
-* __Has the SIRO approved this data movement?__
-
-#### Information assurance
-
-* __Have you discussed the data movement with the relevant information security team(s)?__  
-  The relevant information security teams are:
-    + __MoJ HQ__ -- [cybersecurity team](mailto:cybersecurity@digital.justice.gov.uk)
-    + __HMCTS__ -- [information assurance team](https://intranet.justice.gov.uk/about-hmcts/digital-change/information-assurance-and-data-security/)
-    + __HMPPS__ -- [information management and security team](mailto:informationmgmtsecurity@noms.gsi.gov.uk)
-* __Have you considered all other information assurance requirements, including, but not limited to, those arising under the Public Records Act, the Official Secrets Act, the Data Protection Act and the Freedom of Information Act?__
-
-#### Confirmation
-
-* __Have you read and understood the Analytical Platform guidance and acceptable use policy?__
-
-### Data Minimisation
-
-It is good practice to undertake data minimisation when moving data to the Analytical Platform as all software imposes [memory limits](https://moj-analytical-services.github.io/platform_user_guidance/annexes.html#data-minimisation).
-
-Data minimisation is also a princicple of GDPR. Any personal data used should be:
-
-* adequate;
-* relevant; and
-* limited to what is necessary.
-
-In practice, you may want to consider whether a subset of a larger data set (removing irrelevant fields or observations) is sufficient for your analysis.
-
-## Data Offshoring
 
-The Analytical Platform is hosted on [Amazon Web Services (AWS)](https://aws.amazon.com/) (a US company part of the Amazon group) with primary data storage and processing locations in their [European regions](https://aws.amazon.com/about-aws/global-infrastructure/).
+*   Criminal Injuries Compensation Authority (CICA)
+*   Children and Family Court Advisory and Support Service (CAFCASS)
+*   Criminal Cases Review Commission (CCRC)
+*   Legal Services Board (LSB)
+*   Parole Board for England and Wales
+*   Youth Justice Board for England and Wales (YJB)
+*   Civil Justice Council (CJC)
+*   Family Justice Council (FJC)
+*   Sentencing Council for England and Wales
+*   Office for Legal Complaints (Legal Ombudsman for England and Wales)
+*   The Official Solicitor to the Senior Courts
+*   The Public Trustee
+*   Prisons and Probation Ombudsman (PPO)
+
+#### Moving personal data
+
+If you want to move personal data onto the Analytical Platform, you should check if a [Data Protection Impact Assessment (DPIA)](https://intranet.justice.gov.uk/guidance/knowledge-information/protecting-information/privacy-reform/data-protection-impact-assessments-dpias/) or Privacy Impact Assessment (PIA) already exists for the processing of the personal data. The Information Asset Owner (IAO) of the data should be able to advise you if a DPIA or PIA already exists.
+
+If a DPIA or PIA already exists, you should update the document to reflect use of the Analytical Platform.
```

Comment:
'and make sure that the IAO is aware and approves of this change'? (or is that implicit in the updating of the document? I'm trying to guard against people just adding the detail to a copy that sits on their own area in dom1 and is never seen by DPO)

Reply:
Added in fe48793.